exercises/leap/leap_tests.plt

```diff
@@ -1,19 +1,24 @@
-
 :- begin_tests(leap_tests).
-:- use_module(library(plunit)).
```

Comment:
Do you have to have this line to bring in `plunit`? I do not. I'm on a Mac and CI is Linux.